Add Create Expressions permission (#6120)
hammerandchisel/discord-api-docs: docs/topics/Permissions.md

| MODERATE_MEMBERS \*\*                  | `0x0000010000000000` `(1 << 40)` | Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels |              |
| VIEW_CREATOR_MONETIZATION_ANALYTICS \* | `0x0000020000000000` `(1 << 41)` | Allows for viewing role subscription insights                                                                                                       |              |
| USE_SOUNDBOARD                         | `0x0000040000000000` `(1 << 42)` | Allows for using soundboard in a voice channel                                                                                                      | V            |

> | CREATE_EXPRESSIONS                     | `0x0000080000000000` `(1 << 43)` | Allows members to add custom emojis, stickers and sounds in this server.                                                                            |              |

Reviewer: The naming scheme seems to match the one of the "MANAGE_GUILD_EXPRESSIONS" permission

https://github.com/discord/discord-api-docs/blob/70bc0f05a90b952f376ac2071dba52225dedf7c6/docs/topics/Permissions.md?plain=1#L65
```suggestion
| CREATE_GUILD_EXPRESSIONS               | `0x0000080000000000` `(1 << 43)` | Allows for creating emojis, stickers, and soundboard sounds                                                                                         |              |
```
